Exercise_B/index.html

```diff
@@ -19,16 +21,70 @@
 
   <body>
     <!--Header (flower shop logo block element)-->
+    <nav class="navbar navbar-expand-lg bg-light">
+      <div class="container-fluid">
+        <a class="navbar-brand">
+          <img class="p-0" style="width: 10rem" src="assets/images/logo.png">
+        </a>
+        </div>
+      </div>
+    </nav>
+
+
     <!--Banner Jumbotron of the image-->
+   
+      <div class="bg-image" >
+        
+        <img  class="col-sm-12 img-fluid "  src="/Exercise_B/assets/images/flowershop.jpg">
+
+        <div class="mask text-center d-flex align-items-center justify-content-center">
+        <p class=" text fs-1 text-white p-1"><strong> Beautiful Flowers</strong></p>
+        <button type="button" class="btn btn-success rounded-pill p-1"><strong> Learn more</strong></button>
+        <p class="text position-relative"></p>
+      </div>
+      </div>
 
+     
+       
     <!--Four containers with img / text-->
+
+
     <!-- 4 containers -->
+    <div class="container-fluid bg-info text-center">
+      <div class="row justify-content-evenly">
 
+      <div class="col-lg-3 col-sm-12 col-md-6">
+      <img  src="./assets/images/flower-icon.png" alt="" style="width:70px">
+      <span class="">Flowers for every kind of situation</span>
+      </div>
+      
     <!-- Payments Div -->
+    <div class="col-lg-3 col-sm-12 col-md-6">
+    <img src="./assets/images/payment-icon.png" alt="" style="width: 70px;">
+    <span>All major payments kinds accepted</span>
+    </div>
+
     <!-- Consultation Div -->
+    <div class="col-lg-3 col-sm-12 col-md-6">
+<img src="./assets/images/phone-icon.png" alt="" style="width: 70px" >
+<span>Free consulation</span>
+</div>
 
     <!-- Delivery Div -->
+    <div class="col-lg-3 col-sm-12 col-md-6">
+<img src="./assets/images/truck-icon.png" alt="" style="width: 70px">
+<span>City wide delivery</span>
+      </div>
+    </div>
+  </div>
+
+
+
 
     <!--Scripts required for bootstrap CDN-->
+  
+    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.3/dist/umd/popper.min.js" integrity="sha384-eMNCOe7tC1doHpGoWe/6oMVemdAVTMs2xqW4mwXrXsW0L84Iytr2wi5v2QjrP/xp" crossorigin="anonymous"></script>
+      
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/js/bootstrap.min.js" integrity="sha384-cn7l7gDp0eyniUwwAZgrzD06kc/tftFf19TOAs2zVinnD/C7E91j9yyk5//jjpt/" crossorigin="anonymous"></script>
   </body>
 </html>

```

Comment:
For this exercise, you would need to increase the height on the bottom part where you add in the icons and text and in addition to this, make sure to use Grid or Flex to create something similar to the image.